python/python/glide/async_commands/core.py

```diff
@@ -1322,6 +1322,38 @@ async def zcount(
             ),
         )
 
+    async def zpopmin(
+        self, key: str, count: Optional[int] = None
+    ) -> Mapping[str, float]:
+        """
+        Removes and returns the members with the lowest scores from the sorted set stored at `key`.
+        If `count` is provided, up to `count` members with the lowest scores are removed and returned.
+        Otherwise, only one member with the lowest score is removed and returned.
+
+        See https://redis.io/commands/zpopmin for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            count (Optional[int]): If not specified or None, pops one member.
+            If `count` is higher than the sorted set's cardinality, returns all members and their scores.
+
+        Returns:
+            Map[str, float]: A map of the removed members and their scores, ordered from the one with the lowest score to the one with the highest.
+            If `key` doesn't exist, it will be treated as an empy sorted set and the command returns an empty map.
+
+        Examples:
+            >>> await client.zpopmin("my_sorted_set")
+                {'member1': 10.0}  # Indicates that 'member1' with a score of 10.0 has been removed from the sorted set.
+            >>> await client.zpopmin("my_sorted_set", 2)
+                {'member2': 8.0, 'member3': 7.5}  # Indicates that 'member2' with a score of 8.0 and 'member3' with a score of 7.5 have been removed from the sorted set.
+        """
+        return cast(
+            Mapping[str, float],
+            await self._execute_command(
+                RequestType.ZPopMin, [key, str(count)] if count else [key]
```

Comment:
what happens if count == 0 or a negative value? is it acceptable by redis?

Reply:
count == 0 returns an empty map, negative value will raise an error, Ill add a check